packages/hydrogen-react/src/ShopPayButton.tsx

```diff
@@ -36,11 +37,17 @@ type ShopPayVariantAndQuantities = {
   }>;
 };
 
+type ShopPayChannelAttribution = {
+  /** A string that adds channel attribution to the order. Can be either `headless` or `hydrogen` */
+  channel?: 'headless' | 'hydrogen';
```

Comment:
Is the channel limited to just `headless` or `hydrogen` because that is the only possible values that `<shop-pay-button>` accepts or are the selected values is something that limits to `hydrogen-react` and `hydrogen` scope?